processor/processor.go

```diff
@@ -1391,14 +1391,9 @@ func (proc *HandleT) processJobsForDest(subJobs subJob, parsedEventList [][]type
 					shallowEventCopy.Metadata.DestinationID = destination.ID
 					shallowEventCopy.Metadata.DestinationType = destination.DestinationDefinition.Name
 
-					//TODO: Test for multiple workspaces ex: hosted data plane
-					/* Stream destinations does not need config in transformer. As the Kafka destination config
-					holds the ca-certificate and it depends on user input, it may happen that they provide entire
-					certificate chain. So, that will make the payload huge while sending a batch of events to transformer,
-					it may result into payload larger than accepted by transformer. So, discarding destination config from being
-					sent to transformer for such destination. */
-					if misc.Contains(customDestinations, *destType) {
-						shallowEventCopy.Destination.Config = nil
```

Comment:
As the check on customDestinations is removed, Is there not a need to handle other destination configs apart from Kafka to filter ?

Reply:
We can add fields as when needed I actually checked other custom destinations we didn't have any properties which were large as SSL cert but if we find any we can add it